python/lsst/sdm_schemas/schemas/imsim.yaml

```diff
@@ -6015,7 +6015,608 @@ tables:
+    datatype: double
+    description: Error on the centroid (declination) from the multiband Sersic model fit.
+    fits:tunit: deg
+  - name: sersic_reff_xErr
+    "@id": "#Object.sersic_reff_xErr"
+    datatype: double
+    description: Error on the effective radius (x-axis) from the multiband Sersic model fit.
+    fits:tunit: pixel
+  - name: sersic_reff_yErr
+    "@id": "#Object.sersic_reff_yErr"
+    datatype: double
+    description: Error on the effective radius (y-axis) from the multiband Sersic model fit.
+    fits:tunit: pixel
+  - name: sersic_rhoErr
+    "@id": "#Object.sersic_rhoErr"
+    datatype: double
+    description: Error on ellipse rho (correlation coefficient) from the multiband Sersic model fit.
+    fits:tunit:
+  - name: g_sersicFluxErr
+    "@id": "#Object.g_sersicFluxErr"
+    datatype: double
+    description: Error on the g-band flux from the multiband Sersic model fit.
+    fits:tunit: nJy
+  - name: i_sersicFluxErr
+    "@id": "#Object.i_sersicFluxErr"
+    datatype: double
+    description: Error on the i-band flux from the multiband Sersic model fit.
+    fits:tunit: nJy
+  - name: r_sersicFluxErr
+    "@id": "#Object.r_sersicFluxErr"
+    datatype: double
+    description: Error on the r-band flux from the multiband Sersic model fit.
+    fits:tunit: nJy
+  - name: u_sersicFluxErr
+    "@id": "#Object.u_sersicFluxErr"
+    datatype: double
+    description: Error on the u-band flux from the multiband Sersic model fit.
+    fits:tunit: nJy
+  - name: y_sersicFluxErr
+    "@id": "#Object.y_sersicFluxErr"
+    datatype: double
+    description: Error on the y-band flux from the multiband Sersic model fit.
+    fits:tunit: nJy
+  - name: z_sersicFluxErr
+    "@id": "#Object.z_sersicFluxErr"
+    datatype: double
+    description: Error on the z-band flux from the multiband Sersic model fit.
+    fits:tunit: nJy
+  - name: sersic_sersic_indexErr
+    "@id": "#Object.sersic_sersic_indexErr"
+    datatype: double
+    description: Error on the Sersic profile index parameter from the multiband Sersic model fit.
+    fits:tunit:
+  - name: sersic_n_eval_jac
+    "@id": "#Object.sersic_n_eval_jac"
+    datatype: int
+    description: Number of Jacobian evaluations for the multiband Sersic model fit.
+    fits:tunit:
+  - name: sersic_n_iter
+    "@id": "#Object.sersic_n_iter"
+    datatype: int
+    description: Number of iterations in the non-linear fit for the multiband Sersic model.
+    fits:tunit:
+  - name: sersic_chisq_red
+    "@id": "#Object.sersic_chisq_red"
+    datatype: double
+    description: Reduced chi-squared of the best-fit model parameters (chi divided the number of data points) for the multiband Sersic model.
+    fits:tunit:
+  - name: sersic_unknown_flag
+    "@id": "#Object.sersic_unknown_flag"
+    datatype: boolean
+    description: Flag set for failures with an unexpected or unknown cause for the multiband Sersic model.
+    fits:tunit:
+  - name: sersic_is_parent_flag
```

Comment:
If this and `sersic_not_primary_flag` are redundant with the corresponding `deblend_` and `detect_` flags, I think we can drop them.